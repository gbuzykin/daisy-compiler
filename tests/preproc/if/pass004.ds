#if true
#info "case #1"
#elif true
#info "case #2"
#else
#info "case #3"
#endif

#if true
#info "case #4"
#elif false
#info "case #5"
#else
#info "case #6"
#endif

#if false
#info "case #7"
#elif true
#info "case #8"
#else
#info "case #9"
#endif

#if false
#info "case #10"
#elif false
#info "case #11"
#else
#info "case #12"
#endif
