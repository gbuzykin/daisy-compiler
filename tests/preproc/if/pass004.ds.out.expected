./preproc/if/pass004.ds:2:2: info: "case #1"
 2 | #info "case #1"
   |  ^~~~
./preproc/if/pass004.ds:10:2: info: "case #4"
 10 | #info "case #4"
    |  ^~~~
./preproc/if/pass004.ds:20:2: info: "case #8"
 20 | #info "case #8"
    |  ^~~~
./preproc/if/pass004.ds:30:2: info: "case #12"
 30 | #info "case #12"
    |  ^~~~
./preproc/if/pass004.ds: info: warnings 0, errors 0
